### ISSUE TITLE
BuildSchema derive macro fails when using two newtypes of same underlying type

### ISSUE THREAD

**Issue by ceranco** [Ten0/serde_avro_fast#25]
Hi, when using the `BuildSchema` derive macro on struct with multiple newtypes around the same underlying type, it fails an assertion:

```rust
use serde_avro_derive::BuildSchema;

#[derive(BuildSchema)]
struct A(i32);

#[derive(BuildSchema)]
struct B(i32);

#[derive(BuildSchema)]
struct C(i64);

#[derive(BuildSchema)]
struct Foo {
    a: A,
    b: B,
}

#[derive(BuildSchema)]
struct Bar {
    a: A,
    c: C,
}

println!("{}", Foo::schema().unwrap().json()); // This fails

println!("{}", Bar::schema().unwrap().json()); // This works
```

The error is as follows:
> assertion `left == right` failed
  left: 2
 right: 1

**Comment by Ten0** [comment]
Thanks for opening this issue. I'll fix it in the coming days.

**Comment by Ten0** [comment]
FIx published in [0.3.1](https://github.com/Ten0/serde_avro_fast/commit/cb4368df6ed181ef2b34fd071abb85e780ac35d4).

**Comment by ceranco** [comment]
Awesome, thanks for fixing this so quickly!
